syntax = "proto3";

package euler;

option cc_enable_arenas = true;
option java_outer_classname = "TensorShapeProto";
option java_multiple_files = true;
option java_package = "com.alibaba.euler";
option go_package = "github.com/alibaba/euler/euler/go/core/framework";

message TensorShapeProto {
  repeated int64 dims = 2;
};